build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.7.4-0
```

Comment:
can you make that change here? I'll make sure everything else looks OK and then can push a new image.